### ISSUE TITLE
Crystal language indent not working, (being fixed to 2 space indent)

### ISSUE THREAD

**Issue by greentornado** [Glavin001/atom-beautify#1201]
# Description

The results of beautification are not what I expect.
# Expected Results

The beautified code should have looked like:

```
def route
    before_get "/" do |env|
        env.set "is_kemal_cool", true
    end

    get "/" do |env|
        is_kemal_cool = env.get "is_kemal_cool"
        "Kemal cool = #{is_kemal_cool}"
    end

    get "/foo" do |env|
        env.response.content_type = "application/json"
        {1 => 2, 3 => 4}.to_json
    end
end
```
# Steps to Reproduce
1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!
# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/b627c42b500d91146d6d158f8ecff945
# Checklist
- [x] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] I have reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
  so I know this is not a duplicate issue
- [x] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by Glavin001** [comment]
Looks like there are two issues:
1) Current Crystal beautifier does not support any options however it incorrectly shows it supports all options: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/crystal.coffee#L13 (was `true`, I just fixed to `false`)
2) Crystal beautifier does not support any options, including `indent_size`.

I'll fix problem 1 in next release. I would be happy to review and merge a Pull Request from someone implementing more options for the Crystal beautifier.

**Comment by Glavin001** [comment]
Published to v0.29.17
